helm/flowforge/templates/service-ingress.yaml

```diff
@@ -34,3 +37,9 @@ spec:
             name: forge
             port:
               number: 80
+  {{- if .Values.ingress.certManagerIssuer }}
+  tls:
+  - hosts:
+    - {{ $forgeHostname }}
+    secretName: flowforge-tls
```

Comment:
Like with the broker, I suggest renaming the secret name to the value of the domain name (`{{ $forgeHostname }}` in this case).